src/Findologic/Response/Xml21/Filter/Filter.php

```diff
@@ -211,12 +211,16 @@ private static function parseSubFilters(CategoryFilterValue $filterValue, array
         }
     }
 
-    private static function handleRatingFilter(ApiRangeSliderFilter $filter): RatingFilter
+    private static function handleRatingFilter(ApiRangeSliderFilter $filter): ?RatingFilter
     {
+        $totalRange = $filter->getAttributes()->getTotalRange();
+        if ($totalRange->getMin() === $totalRange->getMax()) {
```

Comment:
Why is `handleRatingFilter` called when min and max are the same? Shouldn't the method call be skipped since there is nothing to handle.

Reply:
This method is called from `\FINDOLOGIC\FinSearch\Findologic\Response\Xml21\Filter\Filter::getInstance`. 

![image](https://user-images.githubusercontent.com/20555112/91851431-6c39d880-ec5f-11ea-8a3b-34ad7aa036cf.png)

Each handler should be responsible for his own filter type.

Reply:
I see then I'm fine with it